llvm/test/Analysis/CostModel/RISCV/insertelement.ll

```diff
@@ -0,0 +1,114 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 3
```

Comment:
Can you precommit the tests so that we can see the diff?  You don't need separate review for that.

Reply:
Pull request consists from 2 commits (pre-commit test and code change), that can be checked separately (by choosing the required one on GitHub's commits tab)